R/markdown.R

```diff
@@ -23,6 +23,10 @@ markdown <- function(path = NULL, ...) {
   )
 
   xml <- xml2::read_html(tmp, encoding = "UTF-8")
+
+  if (!inherits(xml, "xml_node"))
+    stop("cannot build homepage from '", path, "' without content")
```

Comment:
Would you mind rewriting in line with http://style.tidyverse.org/error-messages.html? Thanks!

Reply:
sure, thanks for pointing me to the resource!  I strived to follow the guideline although I was unsure whether to include hint on how to fix.